source/onboarding/person/index.html.md.erb

```diff
@@ -24,7 +24,15 @@ Join the following
 Where `username` is the part of your email before @ (usually firstname.lastname like john.smith)
 These groups will give you access to the VPN, the [Azure Portal](https://portal.azure.com/) and logs.
 
-- Github - join the appropriate team group
+- GitHub Access - submit pull request to [github-management](https://github.com/hmcts/github-management) repository
```

Comment:
I assume you didn't check this locally, formatting is very broken

![image](https://user-images.githubusercontent.com/21194782/111130342-0368dc80-856f-11eb-8b21-25fd14224a92.png)

Read the [acceptable use policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance), if you agree to follow it then ask someone in your team who has access to add you in the [GitHub management](https://github.com/hmcts/github-management) repository.

Reply:
Modified